src/chronify/time_series_mapper_base.py

```diff
@@ -20,37 +22,52 @@ class TimeSeriesMapperBase(abc.ABC):
     """Maps time series data from one configuration to another."""
 
     def __init__(
-        self, engine: Engine, metadata: MetaData, from_schema: TableSchema, to_schema: TableSchema
+        self,
+        engine: Engine,
+        metadata: MetaData,
+        from_schema: TableSchema,
+        to_schema: TableSchema,
     ) -> None:
         self._engine = engine
         self._metadata = metadata
         self._from_schema = from_schema
         self._to_schema = to_schema
-        # self._from_time_config = from_schema.time_config
-        # self._to_time_config = to_schema.time_config
 
-    @abc.abstractmethod
+
+class CheckSchemaMixins:
```

Comment:
@lixiliu What are the chances that we will have map to something other than datetime? If there is any reasonable chance of that, this name should probably include "to datetime."